.github/workflows/run-tests.yml

```diff
@@ -385,7 +385,7 @@ jobs:
         if: ${{ !cancelled() }}
         with:
           name: junit-xml--${{github.run_id}}--${{github.run_attempt}}--integration-test
-          path: .testoutput
+          path: ./.testoutput/junit.*.xml
```

Comment:
Need to be more specific here. This was already including the coverage files.

Reply:
Yes, before it included everything inside the dictionary. Which includes `.xml` as well as `.out` (coverage files). But we don't want coverage files here. So I made it more specific.